transferencia/views.py

```diff
@@ -26,9 +26,10 @@ def post(self, request: Request, agencia: str, num_conta: str):
 
 
 class ConsultaTransferenciaView(APIView):
-    def get(self, id_transferencia: uuid.uuid4):
-        transferencia = consulta_transferencia(id_transferencia=id_transferencia)
+    def get(self, id_transferencia: uuid.uuid4, agencia_origem: str, num_conta_origem: str):
```

Comment:
Aqui tem que aparecer na mesma ordem do que eh colocado la no urls.py.

```suggestion
    def get(self, agencia_origem: str, num_conta_origem: str, id_transferencia: uuid.UUID):
```